deploy/plays/ms.yml

```diff
@@ -1,10 +1,36 @@
 - hosts: ms
   become: yes
   tasks:
+    - name: Create an out file
+      file: path=./logs/ms.txt state=touch
+
+    - name: Clear the file
+      shell : truncate -s 0 ./logs/ms.txt
+
     - name: Build mainserver
       docker_image:
         name: main_server
-        path: ../../main_server
+        path: ./../../main_server
+      register: t1
+
+    - name: Adding line for t1 'stdout'
```

Comment:
considering that the same tasks are being repeated to collect stdout and stderr of each Ansible task, we should definitely explore for ways in which Ansible allows us to call these two tasks automatically. Code duplication is a dangerous trait to have.

Reply:
Thankfully, Ansible has [callback plugins](http://docs.ansible.com/ansible/2.5/plugins/callback.html) which can be called after each task. In fact, to capture stdout and stderr, there are plugins like [debug](http://docs.ansible.com/ansible/2.5/plugins/callback/debug.html) and [stderr](http://docs.ansible.com/ansible/2.5/plugins/callback/stderr.html). Work done via plugins completely eliminates the code duplication.

Reply:
I have tried all of the callback options explained in Ansible docs. None of them work for the debian packages of Ansible.

The most reliable way to get the debug log is to run the ansible-playbook command with `-vvv` option. Hence, it might be better not to pursue any of these callback techniques.

I am inclined to just use the `-vvv` option and not add the different output formatting techniques we have in this PR. The sample output from `-vvv` is not as clean as the one produced by new code in this PR, but it is serviceable.

@GunjanChhablani What do you say to not pursuing this work and just sticking to ansible-playbook option of `-vvv`. If you agree, then we can close this PR, and pursue some other Ansible-related work.